http/src/request/application/get_original_response.rs

```diff
@@ -0,0 +1,67 @@
+use crate::{
+    client::Client,
+    error::Error,
+    request::{PendingOption, Request},
+    routing::Route,
+};
+use twilight_model::{channel::Message, id::ApplicationId};
+
+/// Get an original interaction response.
```

Comment:
This just needs to be changed to be consistent with the client docs
```suggestion
/// Get the original message, by its token.
```